src/main/java/com/educare/unitylend/model/BorrowRequest.java

```diff
@@ -12,12 +10,27 @@
 @NoArgsConstructor
 @AllArgsConstructor
 public class BorrowRequest {
```

Comment:
1. Add @Data on the DataClass, Post that there will be no need to add @Getter and @Setter tag on every variable in the class. 
2. Make the change to have many to many community to borrow request map. In this particular POJO this should have List<communities>